### ISSUE TITLE
Auto list not displaying when typing into the field T899095

### ISSUE THREAD

**Issue by Ogurecher** [DevExpress/testcafe#5322]
### SC Ticket
https://supportcenter.devexpress.com/internal/ticket/details/T899095
 
### Test
```
import { Selector } from 'testcafe';

fixture `skillsForHealth`
    .page `https://drs.ruat.skillsforhealth.org.uk/Administration/Workpatternlist.aspx`;

test('autolist', async t => {
    await t
        .typeText(Selector('#userName'), 'LOGIN FROM SC TICKET')
        .typeText(Selector('#userPassword'), 'PASSWORD FROM SC TICKET')
        .click(Selector('#btnSubmit'))
        .typeText(Selector('#organisationsTxtSearch'), 'skills for health')
        .debug()
        .expect(Selector('#addPattern').exists).ok();
});
```

### Error
```
A JavaScript error occurred on "https://drs.ruat.skillsforhealth.org.uk/Administration/Workpatternlist.aspx".
      Repeat test actions in the browser and check the console for errors.
      If you see this error, it means that the tested website caused it. You can fix it or disable tracking JavaScript errors in TestCafe. To do the latter, enable the "--skip-js-errors" option.
      If this error does not occur, please write a new issue at:
      "https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md".

      JavaScript error details:
      TypeError: Cannot read property 'top' of undefined
          at init._align (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:21:13463)
          at init._position (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:21:12395)
          at init.open (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:21:9617)
          at n (http://172.27.63.129:41853/hammerhead.js:15:4487)
          at value (http://172.27.63.129:41853/hammerhead.js:6:10725)
          at init._listBound (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:23:12905)
          at init.e (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:3:4265)
          at init.trigger (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:10:6897)
          at init.refresh (https://drs.ruat.skillsforhealth.org.uk/js/kendo/kendo.all.min.js:22:15917)
          at init.e (https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js:3:4265)

      Browser: Chrome 83.0.4103.116 / Windows 10
```

**Comment by Ogurecher** [comment]
This happens because we save the result of the first call to `document.getElementsByTagName('body')`, which happens before the <body> element was added to the page. The <body> element does not trigger `element._onElementAdded` which updates the `DOMMutationTracker` version. Thus `document.getElementsByTagName('body')` always returns the stored empty collection.

index.html
```html
<!DOCTYPE html>
<html>
    <head>
        <script>
            window.savedCollection = document.getElementsByTagName('body');
        </script>
    </head>
    <body></body>
</html>
```

test.js
```js
import { ClientFunction } from "testcafe";

fixture `fixture`
    .page `./index.html`;

test('test', async t => {
    const getSavedCollectionLength = new ClientFunction(() => window.savedCollection.length);
    
    await t.expect(getSavedCollectionLength()).eql(0);

    const getNewCollectionLength   = new ClientFunction(() => document.getElementsByTagName('body').length);

    await t.expect(getNewCollectionLength()).eql(1);
});
```

**Comment by MaxStroh** [comment]
@Ogurecher, Thank you for the fast reply on ticket #5502 ! And that the error is already known makes us hope that a fix is in sight. Is it possible to already estimate when a released fix can be roughly expected?

**Comment by Ogurecher** [comment]
@MaxStroh, [The fix](https://github.com/DevExpress/testcafe-hammerhead/pull/2421) is already merged into the master branch of [testcafe-hammerhead](https://github.com/DevExpress/testcafe-hammerhead) and will be included in the next TestCafe version.

We can't share a timeframe as any personal estimate may be misleading, but we usually release new versions every two-three weeks.

**Comment by AndreyBelym** [comment]
This fix was released in `testcafe@1.9.3`. Thank you for your patience 👍
